README.md

```diff
@@ -45,7 +45,7 @@ Sample `.pre-commit-config.yaml`
 
 ## What does it do?
 
-### Separates imports into three sections
+### Separates imports into three sections (stdlib, third party, first party)
```

Comment:
this breaks links, please don't change the anchor

Reply:
Moved it